Modules/Rat_Board.py

```diff
@@ -0,0 +1,31 @@
+from uuid import UUID
+
+
+class RatBoard(object):
+    """
+    A rescue board
+    """
+
+    def __init__(self, handler=None):
+        """
+        Create a new Rescue Board.
+
+        Args:
+            handler ():
+        """
+        self.handler = handler
+        # handler.my_board = self
+
+    def from_api(self, api_id: UUID, create: bool = True, **kwargs) -> None:
```

Comment:
A method from_api(self, rescue: Rescue) would help me cut down on duplicate code. Can you either modify or overload it?

Reply:
Yes, il refactor `RescueBoard.from_api` to accept a `Rescue` object directly.

il see to that today or tomorrow (skoolwork allowing)